src/ORM/DataObject.php

```diff
@@ -3680,4 +3714,103 @@ public function setJoin(DataObject $object, $alias = null)
         }
         return $this;
     }
+
+    /**
+     * Find objects in the given relationships, merging them into the given list
+     *
+     * @param string $source Config property to extract relationships from
+     * @param bool $recursive True if recursive
+     * @param ArrayList $list Optional list to add items to
```

Comment:
is it more accurate to say this?
> list to add to for all items found

Maybe a note that this is used mainly to easily enable recursion?